src/ast/annot.ml

```diff
@@ -0,0 +1,55 @@
+open Format
+
+type annot =
+  { annotid : string
+  ; items : item list
+  }
+
+and item =
+  | Atom of string
+  | String of string
+  | Id of string
+  | Int of string
+  | Float of string
+  | Parens of item list
+  | Annot of annot
```

Comment:
Where does the `Annot of annot` comes from ? I don't see this in the spec, I only see something that matches your `Parens` case.

Reply:
Actually I believe we should use a representation close to S-expressions:

```ocaml
type sexp = Atom of string | List of sexp list
```

WDYT ?

Reply:
After offline discussion: use a representation close to S-expressions

Reply:
https://github.com/janestreet/sexplib0/blob/master/src/sexp.mli